test/welcomePage.spec.js

```diff
@@ -0,0 +1,21 @@
+import { shallowMount, mount } from "@vue/test-utils"
+import AppBody from "../components/AppBody.vue"
+import { BJumbotron } from 'bootstrap-vue'
+
+describe('Welcome page', () => {
+  it('Welcome page should display what you can create', () => {
+    const wrapper = shallowMount(AppBody)
+    expect(wrapper.find("p").text()).toBe("You can create new one or see and edit your current employees")
+  })
+  it('has expected default structure', async () => {
```

Comment:
I would change a little bit name of test - in this specific test case you check if external jumbotron component gets proper properties, it should be visible at first look what test does. So name could be: "used Jumbotron external component gets proper data"